systemd/openqa-trigger-from-obs-update.timer

```diff
@@ -6,4 +6,4 @@ OnCalendar=*:0/3
 Unit=openqa-trigger-from-obs-update.service
 
 [Install]
-WantedBy=default.target
+WantedBy=timers.target
```

Comment:
This makes sense according to https://man.archlinux.org/man/systemd.special.7. We also use this in other timer-triggered units.